### DISCUSSION TITLE
Velocities Frame

### DISCUSSION THREAD

**Discussion by fernandafenelon** [JSBSim-Team/jsbsim#568] [Q&A]
Hello everyone,

I trimmed my aircraft both for negative gamma and positive gamma.
However, for both situation I have a POSITIVE vertical velocities.
I would like to know the sign convention for the velocities. How do I know from the velocities if my aircraft is going up or down?

I tried the properties below.

 fdm['velocities/u-fps']
fdm['velocities/v-fps']
fdm['velocities/w-fps']
      
fdm['velocities/u-aero-fps']
 fdm['velocities/v-aero-fps']
fdm['velocities/w-aero-fps']

Thank you in advance.

**Comment by seanmcleod** [comment]
@fernandafenelon I think you're misunderstanding what the `UVW` velocities represent, and confusing them with inertial velocities in the `NED` local frame of reference.

![image](https://user-images.githubusercontent.com/824856/149475762-114897bc-197b-424a-96e4-67ed7333f920.png)

![image](https://user-images.githubusercontent.com/824856/149477046-b22ed6f8-e4e9-4828-a8ad-bd595b21365c.png)

So the `W` velocity is the velocity of the relative wind along the Z-body axis. With wind from below being positive, i.e. when alpha is positive.

Here is some sample output of the 737 model being trimmed at 1,000ft, 250KIAS for three gamma values, i.e. -5, 0 and +5.

As you'll notice all 3 cases end up with virtually the same alpha value, and hence their `W` velocity is basically identical, and is independent of the flight path angle. 

```sh
-5 gamma

  Trim Results:
       Angle of Attack:   3.18  wdot:  5.56e-05 Tolerance: 1.000000e-03  Passed
              Throttle:   0.31  udot: -1.13e-04 Tolerance: 1.000000e-03  Passed
            Pitch Trim:  -0.20  qdot: -7.61e-11 Tolerance: 1.000000e-04  Passed
           
velocities/w-fps:  23.706067358786495
velocities/h-dot-fps:  -37.2959062187672
velocities/v-down-fps:  37.2959062187672


0 gamma

  Trim Results:
       Angle of Attack:   3.17  wdot:  5.46e-05 Tolerance: 1.000000e-03  Passed
              Throttle:   0.61  udot: -5.10e-05 Tolerance: 1.000000e-03  Passed
            Pitch Trim:  -0.20  qdot: -7.49e-11 Tolerance: 1.000000e-04  Passed

velocities/w-fps:  23.637287307401838
velocities/h-dot-fps:  7.105427357601002e-14
velocities/v-down-fps:  -7.105427357601002e-14


+5 gamma

  Trim Results:
       Angle of Attack:   3.11  wdot:  5.12e-05 Tolerance: 1.000000e-03  Passed
              Throttle:   0.80  udot: -3.79e-05 Tolerance: 1.000000e-03  Passed
            Pitch Trim:  -0.19  qdot: -7.01e-11 Tolerance: 1.000000e-04  Passed

velocities/w-fps:  23.238496740042287
velocities/h-dot-fps:  37.29590687305905
velocities/v-down-fps:  -37.295906873058954
```

Lastly, in terms of the difference between `'velocities/w-fps` and `velocities/w-aero-fps`, the `aero` ones include any wind.

```c++
  // Combine the wind speed with aircraft speed to obtain wind relative speed
  vAeroUVW = in.vUVW - in.Tl2b * in.TotalWindNED;
```
